src/lamplib/tasks/download.py

```diff
@@ -49,27 +47,86 @@ def fetch_and_install(self):
                     logging.info(
                         "On versions of MacOS between 10.12 and 10.14, "
                         "you have to disable System Integrity Protection first. "
-                        "See https://apple.stackexchange.com/a/208481 for instructions")
+                        "See https://apple.stackexchange.com/a/208481 for instructions"
+                    )
                 if minor_ver >= 15:
                     # Instructions derived from https://github.com/NixOS/nix/issues/2925#issuecomment-539570232
                     logging.info(
-                        "On MacOS Catalina and later, use synthetic.conf. "
-                        "The following instructions assume that disk1 is the synthesized AFPS volume of the primary, "
-                        "internal volume your Mac. Verify this assumption with `diskutil list`, substituting the "
-                        "appropriate volume name if required. \n\n"
-
-                        "echo 'data' | sudo tee -a /etc/synthetic.conf\n"
-                        "*Restart your system before continuing*\n\n"
-
-                        "sudo diskutil apfs addVolume disk1 APFSX Data -mountpoint /data \\\n"
-                        "sudo diskutil enableOwnership /data \\\n"
-                        "sudo chflags hidden /data \\\n"
-                        "echo \"LABEL=Data /data apfs rw\" | sudo tee -a /etc/fstab \\\n"
-                        "mkdir /data/mci"
-                        )
+                        f"""
+
+😲 You must create the parent directory {self._name} for the genny toolchain.
+   You are on On MacOS Catalina or later, so use use the synthetic.conf method.
+
+We wish we didn't have to do this.
+
+1️⃣  Step 1 1️⃣
+Run the followng command
+
+    echo 'data' | sudo tee -a /etc/synthetic.conf
```

Comment:
I've just been using a symlink to an existing dir (https://apple.stackexchange.com/a/372500), which has been working for me on Catalina (and Big Sur). It seems a lot easier than simulating a volume 😄

Reply:
oops didn't mean to approve.

Reply:
@guoyr per convo, this is the method I used and lets me build on catalina--seems the symlink method won't work after the build team's catalina updates.

Reply:
Sweet. This is our best option short of a fully static genny build